app/main/views/briefs.py

```diff
@@ -417,7 +417,7 @@ def _render_not_eligible_for_brief_error_page(brief, clarification_question=Fals
             reason = data_reason_slug = "supplier-not-on-lot"
     else:
         reason = "supplier-not-on-framework"
-        data_reason_slug = "supplier-not-on-{}".format(brief['frameworkFramework'])
+        data_reason_slug = "supplier-not-on-{}".format(brief['frameworkSlug'])
```

Comment:
The template uses the `reason`, which remains unchanged, for logic.

The `data_reason_slug` has been changed but is just passed straight through and is then sent to GA as is. I have checked with Ash and this is completely fine.